server/routes/explore/api.py

```diff
@@ -115,7 +115,9 @@ def detect_and_fulfill():
   data_dict = copy.deepcopy(utterance.insight_ctx)
   utterance.prev_utterance = None
   data_dict[Params.CTX.value] = serialize.save_utterance(utterance)
-
+  data_dict[Params.ENABLE_NL_FULFILLMENT] = request.get_json().get(Params.ENABLE_NL_FULFILLMENT, "")
+  data_dict[Params.EXP_MORE_DISABLED] = request.get_json().get(Params.EXP_MORE_DISABLED, "")
```

Comment:
Not sure if you want to use `.value` to be sure?  I'm never sure the enums just work

Reply:
done